maps/CEVEris/_CEV_Eris.dmm

```diff
@@ -54965,6 +54965,9 @@
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers{
 	dir = 6
 	},
+/obj/machinery/vending/hydroseeds{
+	slogan_delay = 700
+	},
```

Comment:
```suggestion
```

Reply:
That's not how it's done... there could be consequences. Just do it with the proper tool.

Reply:
If they can merge Kegdo's untested PRs every two weeks I don't see a problem with this.